window/src/lib.rs

```diff
@@ -264,6 +264,9 @@ pub trait WindowOps {
         Ok(())
     }
 
+    /// Hide other app windows
```

Comment:
I don't think we need to add a method to the window for a function that belongs on the connection; remove this?

Reply:
Yep!